core-data/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/data/ExcludedNames.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.confsched2022.data
+
+internal object ExcludedNames {
+    val LIST = listOf(
```

Comment:
I think there is not so much difference. But If you use setOf(), it is said that the time conplexity will O(1) instead of O(n) when you check contains() because Set uses hash under the hood.

Reply:
@takahirom 
I see! That's interesting because I had no idea about the difference between setOf and listOf in terms of computational complexity!
I spent about 10 minutes searching the web for information describing it, but could not find it.
For the sake of posterity, if you don't mind, could you tell me where you got the quote from?

Reply:
I googled 👀 
![image](https://user-images.githubusercontent.com/1386930/190905933-f5a115d5-ae0f-46c6-8aa6-2c33b219d5d2.png)
https://www.bigocheatsheet.com/

And you can see the benchmark of it.
https://stackoverflow.com/a/10762995/4339442